src/unittests/floating_point.spec.ts

```diff
@@ -3394,41 +3394,71 @@ g.test('roundInterval')
     );
   });
 
-g.test('saturateInterval_f32')
-  .paramsSubcasesOnly<ScalarToIntervalCase>(
-    // prettier-ignore
-    [
-      // Normals
-      { input: 0, expected: 0 },
-      { input: 1, expected: 1.0 },
-      { input: -0.1, expected: 0 },
-      { input: -1, expected: 0 },
-      { input: -10, expected: 0 },
-      { input: 0.1, expected: [kMinusOneULPFunctions['f32'](reinterpretU32AsF32(0x3dcccccd)), reinterpretU32AsF32(0x3dcccccd)] },  // ~0.1
-      { input: 10, expected: 1.0 },
-      { input: 11.1, expected: 1.0 },
-      { input: kValue.f32.positive.max, expected: 1.0 },
-      { input: kValue.f32.positive.min, expected: kValue.f32.positive.min },
-      { input: kValue.f32.negative.max, expected: 0.0 },
-      { input: kValue.f32.negative.min, expected: 0.0 },
+// Need to explicitly coerce expected value to IntervalBounds, because TS
+// doesn't correctly infer the type later.
```

Comment:
Another way that might be better for multiple cases in a trait:
``` typescript
// prettier-ignore
const kSaturateIntervalCases = {
  f32: [
    { input: 0.1, expected: [kMinusOneULPFunctions['f32'](reinterpretU32AsF32(0x3dcccccd)), reinterpretU32AsF32(0x3dcccccd)] }, // ~0.1
  ] as ScalarToIntervalCase[],
  f16: [
    { input: 0.1, expected: [reinterpretU16AsF16(0x2e66), kPlusOneULPFunctions['f16'](reinterpretU16AsF16(0x2e66))] }, // ~0.1
  ] as ScalarToIntervalCase[],
} as const;
```

Reply:
Done